tests/test_stuff.py

```diff
@@ -7,3 +10,22 @@
 
 def test_schedule_systems():
     scheduled_system()
+
+# we've refactored scheduled_system() so that it is decoupled from the items query and now we can
+# individually test each item.  So we need to write a test FOR EACH happy path branch of the
+# original implementation scheduled_systems.
+# I've included a sample of one of how to test part of one of those branches.
+def test_annual_report_uploaded_assigns_outcome():
```

Comment:
To begin with, I'd need to write a test for each line of the original if statement before refactoring to ensure I hadn't broken anything.  We'd need to add database object in order to set up that testing since the function is tied to the ORM.

This is an example of how I would convert those tests to unit tests after we've decoupled the method from the ORM.